mortgage/namespace.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: mortgage-label
+  labels:
+    argocd.argoproj.io/managed-by: openshift-gitops
```

Comment:
will this affect other test cases that rely on mortgage?

since we only want this to be applicable for the git repo test case, should we create a new app instead?

Reply:
I dont think it affects git applications in general it will just keep the namespace binded to argocd even when subscription mortgage is deployed

Reply:
hmm I mean for the normal pull model scenario where we add the label. if we use mortgage for those test cases, then would having this namespace.yaml conflict with that tc?

Reply:
potentially, to be safe ill make a mortgage-pm-argo app for this test